wallet-new/server/Main.hs

```diff
@@ -87,8 +89,10 @@ startEdgeNode :: HasCompileInfo
               -> Production ()
 startEdgeNode WalletStartupOptions{..} = do
   withConfigurations conf $ do
-    (sscParams, nodeParams) <- getParameters
-    actionWithWallet sscParams nodeParams wsoWalletBackendParams
+      when (isDebugMode $ walletRunMode wsoWalletBackendParams) $
+          generateSwaggerDocumentation
```

Comment:
Thanks. This is like it was before though. I wanted to move swagger doc out of Production (as running it takes second or two it seems)

But this would work